typescript/packages/plugins/balmy/src/balmy.plugin.ts

```diff
@@ -0,0 +1,19 @@
+import { type Chain, PluginBase } from "@goat-sdk/core";
+import { BalmyService } from "./balmy.service";
+import { type Token } from "./types";
+
+export type BalmyPluginCtorParams = {
+    tokens?: Token[];
```

Comment:
There is no need for this, it's never used

Reply:
ah yeah , I had kept it there initially as for helper cases